testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -460,6 +472,14 @@ def test_fasta2select_aligned(self):
         assert len(
             sel['mobile']) == 30623, "selection string has unexpected length"
 
+    def test_fasta2select_file(self, tmpdir):
```

Comment:
It looks like a few CI matrix entries are failing this new test because the `clustalw2` program is not available. To deal with that, you can add `@pytest.mark.skipif...` following one of the examples in this file that skips when `clustalw2` is absent.